src/edu-benefits/schema.js

```diff
@@ -286,48 +262,58 @@ module.exports = {
       }
     },
     currentlyActiveDuty: {
-      type: 'boolean'
-    },
-    terminalLeaveBeforeDischarge: {
-      type: 'boolean'
+      type: 'object',
```

Comment:
If this grouping makes things easier on the back end I'm fine with it, but having the fields as individual boolean fields also works for the front end.